paperqa/settings.py

```diff
@@ -631,6 +631,20 @@ class Settings(BaseSettings):
             " router_kwargs key with router kwargs as values."
         ),
     )
+    pair_eval_llm: str = Field(
+        default=CommonLLMNames.GPT_4O.value,
+        description="LLM to use for pairwise evaluation.",
+    )
+    pair_eval_llm_config: dict | None = Field(
+        default=None,
+        description=(
+            "Optional configuration for the pair_eval_llm model. More specifically, it's"
+            " a LiteLLM Router configuration to pass to LiteLLMModel, must have"
+            " `model_list` key (corresponding to model_list inputs here:"
+            " https://docs.litellm.ai/docs/routing), and can optionally include a"
+            " router_kwargs key with router kwargs as values."
+        ),
+    )
```

Comment:
Is it okay adding this to settings?

Reply:
The `Settings` should be general purpose. This `pair_eval_llm`/`LFRQAPairwiseEvalEnv` isn't used for general PaperQA, so no let's not add this here.

Let's just have callers pass this info to `LFRQAPairwiseEvalEnv` directly